.github/workflows/cli.yaml

```diff
@@ -17,7 +17,12 @@ permissions:
     packages: write
 jobs:
     build:
+        name: release deb
```

Comment:
Remove the name, because this build step will not be only for `deb`

Reply:
Fixed !